examples/getting_started/example_config.yaml

```diff
@@ -1,146 +1,247 @@
-modalities_setup:
-  run_mode: FROM_SCRATCH
-  settings: 
-    global_num_seen_samples: 0
+settings:  
+  experiment_id: ${modalities_env:experiment_id}
+  referencing_keys:
+    sample_key: input_ids
+    target_key: target_ids
+  training:
+    global_training_log_interval_in_steps: 48
+    global_checkpointing_interval_in_steps: 48
+    global_evaluation_interval_in_steps: 48
+    global_num_seen_steps: 0
+    do_apply_activation_checkpointing: true
```

Comment:
for this small tutorial, I would suggest setting this to false

Reply:
Fixed in bd8ca46c3f9691dfa710ed5852ceb1334359a33e.